### PR TITLE
Add debian metadata and releasing instructions to the repo

### PR DESCRIPTION
**Pull request by j-rivero**
The PR adds the debian metadata I've been using on my fork for releasing this DART fork. This way we reduce the number of places related to releasing to just this repo. The `gbp.conf` file should take care of telling `git-buildpackage` where to get the sources from.

By now I'm maintaining my own PPA to generate packages from it but this should change in the future.

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    ```
+    git push origin azeey/friction_per_shape_more_params
+    ```
+
+The last command will upload the source package to openrobotics PPA and will create the binaries
+for the selected Ubuntu distribution in all arches (supported architectures can be configured in PPA)
+
+## Step 2: Copy packages to packages.osrfoundation.org
+
+TODO
```
Comment (mjcarroll) [inline RELEASING.md:90]: Is this out of the scope of this PR?

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+**One Ubuntu distribution needs to be released at a time**. The information
+about target distribution (i.e Focal) goes only in the Changelog entry. To
+release multiple distribution, repeat this step 1 changing the changelog entry
+and uploading the new source package to the PPA.
+
+A copy of [this fork](https://github.com/ignition-forks/dart) is required to be in the system.
+
+```
+cd dart
+git checkout azeey/friction_per_shape_more_params
```
Comment (mjcarroll) [inline RELEASING.md]: We're going to update this branch to something more official sounding.

**Review by mjcarroll — COMMENTED** [review]
Overall looks reasonable, going to give it another pass once the branches are updated

File: RELEASING.md (hunk 1 above)
Comment (j-rivero) [reply RELEASING.md:90]: it is. I try to keep PRs somehow not very big.

File: RELEASING.md (hunk 2 above)
Comment (j-rivero) [reply RELEASING.md]: 509055f done.

**Comment by j-rivero** [comment]
It would be great to add a volunteer to the Open Robotics PPA team so we get a couple of things: I'm not the only one there in case that I'm offline and check the process of registering in the Ubuntu One account.

**Comment by mjcarroll** [comment]
> It would be great to add a volunteer to the Open Robotics PPA team so we get a couple of things: I'm not the only one there in case that I'm offline and check the process of registering in the Ubuntu One account.

I have a Ubuntu One account under my personal e-mail, you can add me.

**Review by mjcarroll — APPROVED** [review]

File: RELEASING.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+ 1. Build source package files using gbp
+    ```
+    gbp buildpackage -S
+    ```
+
+#### Upload source package to Ubuntu's PPA
+
+ 1. Upload source package (source package files are located in the parent directory `..`)
+    ```
+    dput ppa:openrobotics/dartsim-openrobotics-testing ../dart6_*_source.changes`
```
Comment (azeey) [inline RELEASING.md]: nit: there's an extra backtick at the end of this command

File: RELEASING.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+and uploading the new source package to the PPA.
+
+A copy of [this fork](https://github.com/ignition-forks/dart) is required to be in the system.
+
+```
+cd dart
+git checkout release-6.10
+```
+
+### Update changelog, push changes
```
Comment (azeey) [inline RELEASING.md]: nit: Does this actually push?

**Review by azeey — APPROVED** [review]
This looks good to me, but I have only tested till the `gbp buildpackage -S` command.

File: RELEASING.md (hunk 4 above)
Comment (j-rivero) [reply RELEASING.md]: ouch, indeed it does not. That is done at the end, I've removed it from the tittle ee6501d

File: RELEASING.md (hunk 3 above)
Comment (j-rivero) [reply RELEASING.md]: eecc817

**Pull request merged**